examples/pom.xml

```diff
@@ -0,0 +1,292 @@
+  </modules>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>8</maven.compiler.target>
+    <maven.compiler.release>8</maven.compiler.release>
+    <maven.version>3.6.0</maven.version>
+
+    <!-- library dependency versions -->
+    <commons.io.version>2.5</commons.io.version>
+    <commonjava.directory.version>0.3.1</commonjava.directory.version>
+    <jackson.version>2.9.10</jackson.version>
+    <jackson.databind.version>2.9.10.3</jackson.databind.version>
+    <jline.version>2.14.6</jline.version>
+    <javax.activation.version>1.2.0</javax.activation.version>
+    <javax.xml.bind.version>2.3.0</javax.xml.bind.version>
+    <jersey.version>2.25</jersey.version>
+    <jib.version>2.3.0</jib.version>
+    <jib.goal>dockerBuild</jib.goal>
+    <maven.assembly.plugin.version>3.1.1</maven.assembly.plugin.version>
+    <maven.compiler.plugin.version>3.7.0</maven.compiler.plugin.version>
+    <maven.dependency.plugin.version>3.1.1</maven.dependency.plugin.version>
+    <maven.deploy.plugin.version>2.8.2</maven.deploy.plugin.version>
+    <maven.enforcer.plugin.version>1.3.1</maven.enforcer.plugin.version>
+    <maven.exec.plugin.version>1.6.0</maven.exec.plugin.version>
+    <maven.failsafe.plugin.version>2.21.0</maven.failsafe.plugin.version>
+    <maven.jar.plugin.version>3.1.2</maven.jar.plugin.version>
+    <maven.jib.plugin.version>1.1.0</maven.jib.plugin.version>
+    <maven.resource.plugin.version>3.1.0</maven.resource.plugin.version>
+    <maven.surefire.plugin.version>2.21.0</maven.surefire.plugin.version>
+    <mockito.version>2.18.3</mockito.version>
+    <version.lib.checkstyle>8.18</version.lib.checkstyle>
+    <version.plugin.checkstyle>2.17</version.plugin.checkstyle>
+    
+    <!-- if necessary, override with -D -->
+    <coherence.group.id>com.oracle.coherence.ce</coherence.group.id>
+    <coherence.version>20.06-SNAPSHOT</coherence.version>
+
+    <helidon.version>1.3.1</helidon.version>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <!-- module dependencies -->
+      <dependency>
+        <groupId>${coherence.group.id}</groupId>
+        <artifactId>coherence</artifactId>
+        <version>${coherence.version}</version>
+      </dependency>
+      
+      <dependency>
+        <groupId>${coherence.group.id}</groupId>
+        <artifactId>coherence-management</artifactId>
+        <version>${coherence.version}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>${coherence.group.id}</groupId>
+        <artifactId>coherence-metrics</artifactId>
+        <version>${coherence.version}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>io.helidon.bundles</groupId>
+        <artifactId>helidon-bundles-webserver</artifactId>
+        <version>${helidon.version}</version>
+      </dependency>
+      
+      <dependency>
```

Comment:
Do we actually need these Jackson dependencies.
We cannot use 2.9.10.3 and need to be at least on 2.10.0.pr1 due to security issues.
I can't see usages in the code so it would be better to just get rid of them. 
Presumably as this is the parent pom and there is only one child module this pom only needs dependencies that the child has so there are a lot of dependencies that can be removed as we only use Coherence and Helidon.

Reply:
Have removed Jackson and all works fine.